src/components/networked-hand-controls.js

```diff
@@ -153,38 +106,269 @@ AFRAME.registerComponent('networked-hand-controls', {
     if (oldData.visible != this.data.visible) {
       this.el.object3D.visible = this.data.visible;     
     }
+
+    // this block is for handling updates() from hand model to controller model, or vice versa.
+    if (oldData.handModelStyle &&
+        oldData.handModelStyle !== this.data.handModelStyle
+      ) {
+      // first, remove old model
+      this.el.removeObject3D(this.str.mesh);
+      ['gltf-model','obj-model'].forEach(modelComponent => {
+        if (this.el.components[modelComponent]) {
+          this.el.components[modelComponent].pause();
+          this.el.components[modelComponent].remove();
+          this.el.removeAttribute(modelComponent);
```

Comment:
Why do you call pause and remove yourself? It should already be done with the removeAttribute.

Reply:
If you look at these controller components, some (all?) of them don't have remove() specified, while the pause() functionality does things that should (also) be done in remove, like remove event listeners. I call pause() to remove those event listeners for those controller components. (The core aframe controller components would ideally be improved, but some of them are quite old and I'm working around them. pausing before removing can't hurt, but it does help.)

Reply:
I see, wasn't aware of those auto-calls underneath, but that makes sense. I'll clean that up to just removeAttribute then and verify that it works as expected.